web-client/iron-svelte-client/README.md

```diff
@@ -1,7 +1,7 @@
 # SvelteKit UI for IronRDP
 
 Web-based frontend using [`SvelteKit`](https://kit.svelte.dev/) and [`Material`](https://material.io) frameworks.
-This is a simple wrapper around the `iron-remote-gui` Web Component demonstrating how to use the API.
+This is a simple wrapper around the `iron-remote-desktop-rdp` Web Component demonstrating how to use the API.
```

Comment:
issue: Needs update. The web component is from `iron-remote-desktop`. You can also mention `iron-remote-desktop-rdp` as the backend. My expectation is that `iron-svelte-client` has a dependency on both `iron-remote-desktop` and `iron-remote-desktop-rdp`.

Reply:
Done